package.json

```diff
@@ -7,5 +7,8 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "marked": "^0.3.5"
```

Comment:
# <a href="#;"> <img title="Critical" alt="Critical" src="https://cdn.sonatype.com/iq-for-scm/1.0/red-bar.png" width="20" height="20"/></a> <a href="#;"> <img title="Direct Dependency" alt="Direct Dependency" src="https://cdn.sonatype.com/iq-for-scm/1.0/d-logo.png" width="20" height="20"/></a> Sonatype IQ found issues introduced by marked : 0.3.5<br /><a href="#;"> <img title="Preview" alt="Preview" src="https://cdn.sonatype.com/iq-for-scm/1.0/Preview.svg" width="70" height="20"/></a>
<details>
<br/>

Threat Level: <strong>Critical (9)</strong> \| [View Component Details in Sonatype Lifecycle](http://tt2.nexus.local:8070/ui/links/application/app-node-public/report/55aa40527c8242709d18bdad63ad91ab/componentDetails/4113a15ac5d7bca158a5?source=pr-line-commenting&tab=violations&utm_source=github)

## :shield: Recommendation
**Bumping to version 4.0.10** will resolve all policy violations for this component (as of _Oct 16, 2024_)

## :page_facing_up: Security Issue Details

<details>
  <summary title="View all (10)">
    View all (10)
  </summary>
  <p></p>

| **Severity** | **Issue** | **Organization Policy Violation** |
| --- | --- | --- |
| <b>Threat level:</b> 9<br><b>CVSS Score:</b> 7.5<br><br><a href="#;"> <img title="Sonatype Deep Dive" alt="Sonatype Deep Dive" src="https://cdn.sonatype.com/iq-for-scm/1.0/DeepDive.svg" width="900" /></a> | [sonatype-2017-0636] | [View Details](http://tt2.nexus.local:8070/ui/links/policyViolationReport/9aba610839ba459eb1e254d65da76ef9?utm_source=github) <br /><img src="https://cdn.sonatype.com/iq-for-scm/1.0/Filler.svg" width="600" height="0" display="hidden">|
| <b>Threat level:</b> 9<br><b>CVSS Score:</b> 7.5<br><br><a href="#;"> <img title="Sonatype Deep Dive" alt="Sonatype Deep Dive" src="https://cdn.sonatype.com/iq-for-scm/1.0/DeepDive.svg" width="900" /></a> | [sonatype-2017-0367] | [View Details](http://tt2.nexus.local:8070/ui/links/policyViolationReport/dd6e01601cea48c1b28f866eb9a5e638?utm_source=github) <br /><img src="https://cdn.sonatype.com/iq-for-scm/1.0/Filler.svg" width="600" height="0" display="hidden">|
| <b>Threat level:</b> 9<br><b>CVSS Score:</b> 7.5<br><br><a href="#;"> <img title="Sonatype Deep Dive" alt="Sonatype Deep Dive" src="https://cdn.sonatype.com/iq-for-scm/1.0/DeepDive.svg" width="900" /></a> | [CVE-2015-8854] The marked package before 0.3.4 for Node.js allows attackers to cause a denial of service (CPU consumption) via unspecified vectors that trigger a "catastrophic backtracking issue for the em inline rule," aka a "regular expression denial of service (ReDoS)." | [View Details](http://tt2.nexus.local:8070/ui/links/policyViolationReport/f626b564b66545028b96932cc1950461?utm_source=github) <br /><img src="https://cdn.sonatype.com/iq-for-scm/1.0/Filler.svg" width="600" height="0" display="hidden">|
| <b>Threat level:</b> 9<br><b>CVSS Score:</b> N/A | None | [View Details](http://tt2.nexus.local:8070/ui/links/policyViolationReport/ce1756f5bdfd41aca9543069f2d6ebed?utm_source=github) <br /><img src="https://cdn.sonatype.com/iq-for-scm/1.0/Filler.svg" width="600" height="0" display="hidden">|
| <b>Threat level:</b> 7<br><b>CVSS Score:</b> 6.5<br><br><a href="#;"> <img title="Sonatype Deep Dive" alt="Sonatype Deep Dive" src="https://cdn.sonatype.com/iq-for-scm/1.0/DeepDive.svg" width="900" /></a> | [sonatype-2018-0076] | [View Details](http://tt2.nexus.local:8070/ui/links/policyViolationReport/2caedfcd315644f69f6506ac78160084?utm_source=github) <br /><img src="https://cdn.sonatype.com/iq-for-scm/1.0/Filler.svg" width="600" height="0" display="hidden">|
| <b>Threat level:</b> 7<br><b>CVSS Score:</b> 6.5<br><br><a href="#;"> <img title="Sonatype Deep Dive" alt="Sonatype Deep Dive" src="https://cdn.sonatype.com/iq-for-scm/1.0/DeepDive.svg" width="900" /></a> | [sonatype-2018-0378] | [View Details](http://tt2.nexus.local:8070/ui/links/policyViolationReport/f8366aa0a64a48458405db990ec9380c?utm_source=github) <br /><img src="https://cdn.sonatype.com/iq-for-scm/1.0/Filler.svg" width="600" height="0" display="hidden">|
| <b>Threat level:</b> 7<br><b>CVSS Score:</b> 6.1<br><br><a href="#;"> <img title="Sonatype Deep Dive" alt="Sonatype Deep Dive" src="https://cdn.sonatype.com/iq-for-scm/1.0/DeepDive.svg" width="900" /></a> | [sonatype-2017-0008] | [View Details](http://tt2.nexus.local:8070/ui/links/policyViolationReport/5327f9a44580498b96daf49885b8ca4b?utm_source=github) <br /><img src="https://cdn.sonatype.com/iq-for-scm/1.0/Filler.svg" width="600" height="0" display="hidden">|
| <b>Threat level:</b> 7<br><b>CVSS Score:</b> 6.1<br><br><a href="#;"> <img title="Sonatype Deep Dive" alt="Sonatype Deep Dive" src="https://cdn.sonatype.com/iq-for-scm/1.0/DeepDive.svg" width="900" /></a> | [sonatype-2017-0637] | [View Details](http://tt2.nexus.local:8070/ui/links/policyViolationReport/5d072aa4ebc546859b18b13b58762c2c?utm_source=github) <br /><img src="https://cdn.sonatype.com/iq-for-scm/1.0/Filler.svg" width="600" height="0" display="hidden">|
| <b>Threat level:</b> 7<br><b>CVSS Score:</b> 6.1<br><br><a href="#;"> <img title="Sonatype Deep Dive" alt="Sonatype Deep Dive" src="https://cdn.sonatype.com/iq-for-scm/1.0/DeepDive.svg" width="900" /></a> | [sonatype-2016-0030] | [View Details](http://tt2.nexus.local:8070/ui/links/policyViolationReport/85edfd60a9c34e18a2b8ddc380373791?utm_source=github) <br /><img src="https://cdn.sonatype.com/iq-for-scm/1.0/Filler.svg" width="600" height="0" display="hidden">|
| <b>Threat level:</b> 7<br><b>CVSS Score:</b> 6.1<br><br><a href="#;"> <img title="Sonatype Deep Dive" alt="Sonatype Deep Dive" src="https://cdn.sonatype.com/iq-for-scm/1.0/DeepDive.svg" width="900" /></a> | [sonatype-2017-0638] | [View Details](http://tt2.nexus.local:8070/ui/links/policyViolationReport/670fb8989c434ec5b867bf556938deca?utm_source=github) <br /><img src="https://cdn.sonatype.com/iq-for-scm/1.0/Filler.svg" width="600" height="0" display="hidden">|
</details>
</details>